tests/CLI/modules/bandwidth_tests.py

```diff
@@ -56,6 +56,16 @@ def test_create_bandwidth(self):
         self.assertEqual('NewRegion', json_output['Name Pool'])
         self.assertEqual('SJC/DAL/WDC/TOR/MON', json_output['Region'])
 
+    def test_edit_bandwidth(self):
+        result = self.run_command(['bandwidth', 'pools-edit', '123456', '--name=MexRegionEdited'])
+        self.assert_no_fail(result)
+        self.assert_called_with('SoftLayer_Network_Bandwidth_Version1_Allotment', 'editObject')
+        json_output = json.loads(result.output)
+        pp(json_output)
```

Comment:
Hi @edsonarios 

Can you please **provide** the **answers** to the following **questions**.

**Questions:**
1. What is pp() , line number: 64?
2. Why pp() is required, line number: 64?

Thanks,
Ramkishor Chaladi.

Reply:
pp it's to format better the output
only is required in develop time, now was removed